ros/src/actuation/vehicles/packages/ymc/node/g30esli_interface/g30esli_ros.cpp

```diff
@@ -0,0 +1,329 @@
+/*
+ * Copyright 2017-2019 Autoware Foundation. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "g30esli_ros.h"
+
+G30esliROS::G30esliROS()
+  : alive_(0), reset_command_(false), stop_keep_secs_(1.0), reset_keep_secs_(0.6), disable_reset_secs_(3.0)
```

Comment:
`stop_keep_secs_` needs to be a little bigger.
When we tested online, at 1.0 sec we saw behavior that stopped immediately after the launch.
Changing this from 1.0 to 2.0 seconds will eliminate this anomalous behavior.
```suggestion
  : alive_(0), reset_command_(false), stop_keep_secs_(2.0), reset_keep_secs_(0.6), disable_reset_secs_(3.0)
```

Reply:
Thanks!
NOTE: I think they're not needed to modify for each vehicle after this PR because of these are common parameters between golfcar.